carma_wm/include/carma_wm/WorldModelUtils.h

```diff
@@ -0,0 +1,111 @@
+#pragma once
+
+/*
+ * Copyright (C) 2021 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include <exception>
+#include <lanelet2_core/LaneletMap.h>
+#include <lanelet2_core/primitives/Area.h>
+#include <lanelet2_core/primitives/Point.h>
+#include <lanelet2_core/primitives/Lanelet.h>
+#include <lanelet2_core/primitives/Polygon.h>
+#include <lanelet2_core/geometry/Polygon.h>
+#include <lanelet2_core/geometry/LineString.h>
+#include <lanelet2_core/utility/Optional.h>
+#include "TrackPos.h"
+#include <boost/geometry.hpp>
+#include <boost/geometry/geometries/polygon.hpp>
+#include <ros/ros.h>
+#include <carma_wm/Geometry.h>
+
+namespace carma_wm
+{
+/**
+ * \brief carma_wm::utils namespace contains implementations for utility read or write functions for stand-alone lanelet map without rest of the CARMAWorldModel features.
+ *        Currently mainly WMBroadcaster (carma_wm_ctrl) is using this to manipulate its own map without creating instance of carma_wm
+ */
+namespace utils
```

Comment:
rather than having the getLaneletsFromPoint method under a generic "utils" namespace maybe giving the namespace a more descriptive name would make them easier to find. 
For example carma_wm::matching::getLaneletsFromPoint or carma_wm::query::getLaneletsFromPoint

Reply:
added query namespace for getLaneletsFromPoint and nonConnectedAdjacentLeft